batchflow/models/tf/mobilenet.py

```diff
@@ -219,8 +222,203 @@ def block(cls, inputs, filters, residual=False, strides=1, expansion_factor=6, w
         tf.Tensor
         """
         num_filters = int(cls.num_channels(inputs, kwargs.get('data_format')) * expansion_factor * width_factor)
-        conv_filters = [num_filters, num_filters, filters]
-        x = conv_block(inputs, 'cna Cna cn', conv_filters, [1, 3, 1], name=name, strides=[1, strides, 1], **kwargs)
+
+        x = conv_block(inputs, 'cna', num_filters, kernel_size=1, name='%s-exp' % name, strides=1, **kwargs)
+        x = depthwise_conv(x, kernel_size=3, strides=strides, padding='same',
+                           data_format=kwargs['data_format'], name='%s-depthwise' % name)
+        x = conv_block(x, 'nacn', filters, kernel_size=1, strides=1, name='%s-down' % name, **kwargs)
```

Comment:
What's the point in splitting 1 call of `conv_block` into 3 calls?

Reply:
because C performs depthwise+pointwise when only depthwise is needed. We don't have a letter for depthwise_conv only.

Reply:
That's not true. Our layers perform pw-conv only if needed. And there is tf layer for dw-only conv. So we can easily add a letter for that.

Reply:
done.